vsts/pipelines/_buildimage-bases.job-template.yml

```diff
@@ -0,0 +1,39 @@
+parameters:
+  name: ''
+  displayName: ''
+  imageDir: ''
+
+
+jobs:
+- job: ${{ parameters.name }}
+  displayName: ${{ parameters.displayName }}
+  steps:
+
+  - task: ShellScript@2
+    displayName: Build images
+    inputs:
+      scriptPath: ./build/build-buildimage-bases.sh
+      args: ${{ parameters.imageDir }}
+
+  - task: CopyFiles@2
+    displayName: Copy artifacts to staging directory
+    inputs:
+      sourceFolder: '$(Build.SourcesDirectory)/artifacts'
+      contents: '**/*.*'
+      targetFolder: $(Build.ArtifactStagingDirectory)
+      overWrite: true
+    condition: true
+
+  - task: Docker@1
+    displayName: Push built images to dev ACR
+    inputs:
+      command: push
+      azureSubscriptionEndpoint: $(ascName)
+      azureContainerRegistry: $(acrName)
+      pushMultipleImages: true
+      imageNamesPath: '$(Build.ArtifactStagingDirectory)/images/build-image-bases.txt'
+
+  - task: PublishBuildArtifacts@1
```

Comment:
Let's add a docker cleanup task too here as we want to untag the image having the build number(as otherwise those images will stay on the agent consuming space). You could copy the task from our other yml files